lib/ansible/modules/crypto/openssl_csr.py

```diff
@@ -455,7 +471,6 @@ def main():
         ),
         add_file_common_args=True,
         supports_check_mode=True,
-        required_one_of=[['commonName', 'subjectAltName']],
```

Comment:
By removing that line (and not adding subject to it), we allow the subject field to be completly blank. The module will work anyway (so technically it doesn't seem wrong) and openssl will read it happily. But a certificate with a `null` subject has no interest, so should we enforce at least basic CN via anyway the interface allows it ?

Reply:
There could be use cases for certificates with no subject and they are technically valid, so they should be possible to be generated/checked by these modules.

Reply:
I don't know such use case for it. But since openssl doesn't complain it is safe to assume it is valid